src/Week5/HW5.hs

```diff
@@ -0,0 +1,179 @@
+{-# OPTIONS_GHC -Wall #-}
```

Comment:
I checked the assignment pdf and see they're suggesting using cabal install aeson etc to get aeson and bytestring dependencies.
Let's do it in proper stack way (since we're using stack in other projects too).

You can just modify [this line](https://github.com/merrymeerkat/CIS194/blob/142b2189e36fe579d73b726eb710262ebef8200f/CIS194.cabal#L35)

to look like
```
  base >=4.7 && <5, aeson, bytestring
```

Add the Week5.HW5 and Week5.Parser modules to cabal file as you did in previous weeks' solutions
and that should make `stack build` work.

Reply:
makes sense, thanks! I've changed the cabal file :)

Reply:
fixed the week4 confusion